__tests__/reviews.test.js

```diff
@@ -19,7 +19,6 @@ test('should return an array of review objects', () => {
       .expect(200)
       .then(({body : {reviews}}) => {
         reviews.forEach((review) => {
-          expect(Object.keys(review).length).toBe(9);
```

Comment:
What was the reason this was removed? Have a test to make sure the array is populated is important to ensure the test does not pass if given an empty array.